include/pgduckdb/pgduckdb_planner.hpp

```diff
@@ -14,3 +14,4 @@ extern bool duckdb_explain_ctas;
 PlannedStmt *DuckdbPlanNode(Query *parse, const char *query_string, int cursor_options, ParamListInfo bound_params,
                             bool throw_error);
 duckdb::unique_ptr<duckdb::PreparedStatement> DuckdbPrepare(const Query *query, const char *explain_prefix = NULL);
+bool IsAllowedPostgresInsert(Query *query, bool throw_error = false);
```

Comment:
Seems like you don't have https://editorconfig.org/ installed in your editor. Could you do that and re-save this file to add a trailing newline.

Reply:
sure